02-alouette/alouette_test.rb

```diff
@@ -0,0 +1,117 @@
+gem 'minitest', '>= 5.0.0'
+require 'minitest/spec'
+require 'minitest/reporters'
+require 'minitest/autorun'
+require_relative 'alouette'
+
+Minitest::Reporters.use!
+
+class AlouetteTest < Minitest::Test
+  describe "lines_for_verse" do
+    it "returns an array of strings" do
+      lines = Alouette.lines_for_verse(3)
+
+      lines.must_be_kind_of Array
```

Comment:
The style of these tests is not the same as the other weekend warrior tests. I think we would only want to use "must" if we are planning on changing all of them from "assert"